cmd/builder/test/core.builder.yaml

```diff
@@ -1,20 +1,20 @@
 dist:
   module: go.opentelemetry.io/collector/builder/test/core
-  otelcol_version: 0.94.0
+  otelcol_version: 0.97.0
```

Comment:
No need to update this on this PR, we can file a separate PR for updating these versions

Reply:
These tests fail without this update due to the strict versioning checks.

Reply:
Ah, fair enough then :)